### PR TITLE
OWB-1389 Remove destroyed instance from memory

### PR DESCRIPTION
**Pull request by BuZZ-dEE**
Avoids memory leak

### CODE REVIEW

File: webbeans-tomcat7/src/main/java/org/apache/webbeans/web/tomcat7/TomcatInstanceManager.java [hunk 1]
```diff
@@ -60,10 +60,15 @@ public void destroyInstance(Object instance) throws IllegalAccessException, Invo
             }
             catch (Exception e)
             {
-                log.error("Erros is occured while destroying the OpenWebBeans injector instance", e);
+                log.error("Error is occured while destroying the OpenWebBeans injector instance", e);
             }
         }
         this.processor.destroyInstance(instance);
+        this.objects.remove(instance);
```
Comment (rmannibucau) [inline webbeans-tomcat7/src/main/java/org/apache/webbeans/web/tomcat7/TomcatInstanceManager.java:67]: guess it can replace the get?

Comment (BuZZ-dEE) [reply webbeans-tomcat7/src/main/java/org/apache/webbeans/web/tomcat7/TomcatInstanceManager.java:67]: yes, you are right. should i do it?

Comment (rmannibucau) [reply webbeans-tomcat7/src/main/java/org/apache/webbeans/web/tomcat7/TomcatInstanceManager.java:67]: go ahead while you're at it ;)

Comment (BuZZ-dEE) [reply webbeans-tomcat7/src/main/java/org/apache/webbeans/web/tomcat7/TomcatInstanceManager.java]: ok, replaced get.

**Review by struberg — APPROVED** [review]

**Comment by struberg** [comment]
txs for the patch!

**Comment by BuZZ-dEE** [comment]
@struberg i think, this is also useful for the master branch https://github.com/apache/openwebbeans/pull/38

**Pull request merged**